var/spack/repos/builtin/packages/py-fiona/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFiona(PythonPackage):
+    """Alternative Python binding for OGR"""
+
+    homepage = "http://toblerity.org/fiona/"
+    url      = "https://github.com/Toblerity/Fiona/archive/1.7.12.tar.gz"
+
+    version('1.7.12', 'a2269acf64d9c87482e5d67dae19501c')
+
+    depends_on('python@2.7:')
```

Comment:
I don't see this version requirement. In fact, the `setup.py` contains special logic to make things work with 2.6.
```suggestion
    depends_on('python@2.6:', type=('build', 'run'))
```